cli/js/web/workers.ts

```diff
@@ -133,6 +205,12 @@ export class WorkerImpl extends EventTarget implements Worker {
           const message = decodeMessage(new Uint8Array(event.data));
           this.onmessage({ data: message });
```

Comment:
MessageEvent should be passed as an argument to this callback as well

Reply:
sorry for the long wait, so like 
```suggestion
          this.onmessage(new MessageEvent("message", {data: message}));
```
?
should i change the type of `onmessage` param as well?

Reply:
Yes, bit create only a single `new MessageEvent`

Reply:
as in
```ts
if (type === "msg") {
  const message = decodeMessage(new Uint8Array(event.data));
  const ev = new MessageEvent("message", {
    cancelable: false,
    data: this.onmessage ? message : event.data
  });

  this.dispatchEvent(ev);
  continue;
}
```
?